### PR TITLE
Update linguist settings

### PR DESCRIPTION
**Pull request by veteran29**
**When merged this pull request will:**
- title
- improves how GitHub counts language usage in the repository
- https://github.com/github/linguist/blob/master/docs/overrides.md

`docker run -v %cd%:/pwd -w /pwd inloopx/linguist linguist`
Before:
```
55.76%  C++
37.61%  SQF
2.51%   C
1.84%   PAWN
1.76%   Python
0.28%   SourcePawn
0.22%   CMake
0.00%   Batchfile
```
After
```
65.69%  SQF
29.67%  C++
2.59%   C
1.82%   Python
0.23%   CMake
0.00%   Batchfile
```

### CODE REVIEW

**Review by PabstMirror — APPROVED** [review]
neat

File: .gitattributes [hunk 1]
```diff
@@ -2,3 +2,13 @@
 *.png binary
 *.jpg binary
 *.paa binary
+
+# Change GitHub language categorisation
```
Comment (PabstMirror) [inline .gitattributes:6]:
```suggestion
# Change GitHub language categorization
```

**Comment by jonpas** [comment]
Nice! But is config SQF? :thinking:

**Comment by veteran29** [comment]
> Nice! But is config SQF? 🤔

Yeah it's not but it's closer to SQF than it's to CPP. I'm not aware about any other lang we could categorize it with.

**Pull request merged**